web/controllers/rules/pr/merge_conflict.ex

```diff
@@ -6,9 +6,18 @@ defmodule Dwylbot.Rules.PR.MergeConflict do
   @github_api Application.get_env(:dwylbot, :github_api)
   @rule_name "pr_merge_conflicts"
 
-  def apply?(payload) do
-    payload["action"] == "closed" && payload["pull_request"]["merged"]
-  end
+  @doc """
+  iex>apply?(%{"action" => "closed", "pull_request" => %{"merged" => false}})
+  false
+  iex>apply?(%{"action" => "labeled", "label" => %{"name" => "awaiting-review"}})
+  true
+  iex>apply?(%{"action" => "unlabeled"})
+  false
+  """
+
+  def apply?(%{"action" => "closed", "pull_request" => %{"merged" => merged}}), do: merged
+  def apply?(%{"action" => "labeled", "label" => %{"name" => "awaiting-review"}}), do: true
```

Comment:
The trigger of the rule is correct (a user adds the label awaitiing-review on a PR) and the action it to add a comment to let the user that the PR has a merge conflict. However I think this rule should be defined in its own file. This current file deal with a different trigger and I think is a different  rule:
- trigger: a user merge a PR (action closed and pull request status "merged" see line 18
- action: add a comment on **all** the other PR who have a merge conflict.